getting-started-gradle/gradle.properties

```diff
@@ -0,0 +1,6 @@
+#Gradle properties
+quarkusPluginId=io.quarkus
+quarkusPluginVersion=1.8.3.Final
```

Comment:
Hum, normally development use 999-SNAPSHOT.
But I don't know how it works for gradle. @gsmet @aloubyansky any idea?

Reply:
As long as Gradle can find the snapshot (i.e. configured with the proper repo) it should work with 999-SNAPSHOT.

Reply:
Yes my bad, I only updated the `pom.xml`, I will update the `gradle.properties` too.